packages/vat-data/test/test-durable-classes.js

```diff
@@ -105,4 +108,38 @@ test('test defineDurableFarClassKit', t => {
   makeCounterKit('str');
 });
 
+test('finish option', t => {
```

Comment:
hm. not sure I grok this test.

Maybe this disqualifies me as a reviewer? maybe not?